service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/history/SealedOrderHistoryServiceTest.java

```diff
@@ -94,6 +96,7 @@
     private final SealedOrderHistoryService underTest = new SealedOrderHistoryService(
         identityService,
         childrenService,
+        appointedGuardianService,
```

Comment:
missing  exercising the test coverage for this one new mock

Reply:
added a test for new field